test_conformance/compiler/test_feature_macro.cpp

```diff
@@ -686,9 +686,7 @@ int test_consistency_c_features_list(cl_device_id deviceID,
     sort(vec_to_cmp.begin(), vec_to_cmp.end());
     sort(vec_device_feature_names.begin(), vec_device_feature_names.end());
 
-    cl_bool result =
-        std::equal(vec_device_feature_names.begin(),
-                   vec_device_feature_names.end(), vec_to_cmp.begin());
+    cl_bool result = vec_device_feature_names == vec_to_cmp;
     if (result)
```

Comment:
I'm not going to lose sleep over this, but I'd find this check easier to understand without the extra boolean variable (and `operator==` doesn't return a `cl_bool` anyhow).

Reply:
Good point, I've changed it.